Add content for the new building module - Closes #754 (#755)
LiskHQ/lisk-docs: modules/building/pages/ui.adoc

These pages provide a representative sample and kitchen sink of content from the real site.

To build the UI and preview it in a local web server, run the `preview` command:

 $ gulp preview

You'll see a URL listed in the output of this command:

....
[12:00:00] Starting server...
[12:00:00] Server started http://localhost:5252
[12:00:00] Running server
....

Navigate to this URL to preview the site locally.

While this command is running, any changes you make to the source files will be instantly reflected in the browser.
This works by monitoring the project for changes, running the `preview:build` task if a change is detected, and sending the updates to the browser.

Press kbd:[Ctrl+C] to stop the preview server and end the continuous build.

=== Package for Use with Antora

If you need to package the UI so you can use it to generate the documentation site locally, run the following command:

 $ gulp bundle

If any errors are reported by lint, you'll need to fix them.

When the command completes successfully, the UI bundle will be available at [.path]_build/ui-bundle.zip_.
You can point Antora at this bundle using the `--ui-bundle-url` command-line option.

> If you have the preview running, and you want to bundle without causing the preview to be clobbered, use:

Reviewer: ```suggestion
If you have the preview running, and you want to bundle without causing the preview to be clobbered, use:
```
Am not sure 'clobbered' is the right word here..? Do you mean damaged or impaired..?

 $ gulp bundle:pack

The UI bundle will again be available at [.path]_build/ui-bundle.zip_.

==== Source Maps

The build consolidates all the CSS and client-side JavaScript into combined files, [.path]_site.css_ and [.path]_site.js_, respectively, in order to reduce the size of the bundle.
{url-source-maps}[Source maps] correlate these combined files with their original sources.

This "`source mapping`" is accomplished by generating additional map files that make this association.
These map files sit adjacent to the combined files in the build folder.
The mapping they provide allows the debugger to present the original source rather than the obfuscated file, an essential tool for debugging.

> In preview mode, source maps are enabled automatically, so there's nothing you have to do to make use of them.

Reviewer: ```suggestion
In preview mode, source maps are enabled automatically, so no other requirements are necessary to use them.
```
